### DISCUSSION TITLE
Regarding nzbget

### DISCUSSION THREAD

**Discussion by tathastu871** [termux/termux-packages#8682] [Q&A] [answered]
ERROR   Could not resolve hostname my.newsserver.com: Error 7 - No address associated with hostname

First time using usenet. I have nzb file when i use nzbget file.nzb i get above errors.
Can anyone help with configuration
@agnostic-apollo 
Help needed

**Comment by agnostic-apollo** [comment, marked as answer]
> my.newsserver.com

That is apparently not a real address and just an example. I don't use `nzbget`, so can't help you and @termux does not provide support for external tools. Check its docs at https://nzbget.net/documentation and if still can't figure it out, then best ask in https://github.com/nzbget/nzbget or possibly https://www.reddit.com/r/usenet.
